### DISCUSSION TITLE
What document can prove that I am a Chinese student?

### DISCUSSION THREAD

**Discussion by WRTY1** [community/community#53330] [GitHub Education] [answered]
Last year, my request was approved based on an acceptance letter. However, this year, I have been rejected more than 20 times in the past three weeks. In the most recent rejection, I was asked to upload a different document to demonstrate my current academic status.

Initially, I was rejected for providing incorrect GitHub profiles, but I made the necessary edits. However, when I tried again, I received a message stating that the document could not be used. Unfortunately, the only official documents I have in English are a few provided by my university.
> Like this
> Please upload a different document to demonstrate your current academic status.

When I tried to upload a receipt in Chinese, I received a message saying that the image did not appear to contain my school name. Although I included multiple documents in the image, such as my transcript, syllabus, schedule, and CHSI verification, along with an English translation, I was still informed that my GitHub profile did not have enough information and did not include my school name. Additionally, my student ID does not have any relevant data.
> Like this
> The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible. For example, if your original document is not in English then you might take a photo of the original next to an English translation.

I have exhausted all the possible forms of verification I can obtain at the moment, yet I have been rejected once more. I am unsure of what document a Chinese student can provide to prove their enrollment status. Could you please provide me with some guidance on this matter?

**Comment by samus-aran** [comment, marked as answer]
Hi @WRTY1 

Let's see how we can assist. ⭐ 

I know this may seem counter productive but lets start with the basics. Let's start with your original Chinese evidence and pre-translate it, preferably without image manipulation.

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information. Look at your evidence and pick out the important elements: dates, school name, your name (if. non English), course title etc.
- Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Let us know how you get on 👍

**Reply by whitewatercn** [reply]
> It is simple to write the English translation next to the material and then upload the original text with the translation by taking a photo.For more, you check out [my article](https://www.luogu.com.cn/blog/pnpn111/github-student-packet).


刚在discussion里翻到的前人经验，实测有用！在此基础上做了一些补充 https://xlog.whitewater.wang/Github_Student_Developer_Pack
I just turned over in the discussion is useful! and i make some supplyment based on your post https://xlog.whitewater.wang/Github_Student_Developer_Pack

**Reply by WRTY1** [reply]
Thanks a lot for reverify my document.And now it shows that I was approved.Should I update a new one to reverify?
